packages/shared/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@repo/shared",
+  "version": "0.0.0",
+  "private": true,
+  "main": "index.ts",
+  "types": "index.ts",
```

Comment:
You can remove this `types` field as the package will never be built and will always be evaluated by the parent